Alicerce/Sources/Persistence/SiblingContextCoreDataStack.swift

```diff
@@ -0,0 +1,128 @@
+            backgroundContext.mergePolicy = mergePolicy
+        } else {
+            let workCoordinator = NestedContextCoreDataStack
+                .persistentStoreCoordinator(withType: storeType,
+                                            storeName: storeName,
+                                            managedObjectModel: managedObjectModel,
+                                            shouldAddStoreAsynchronously: shouldAddStoreAsynchronously,
+                                            shouldMigrateStoreAutomatically: shouldMigrateStoreAutomatically,
+                                            shouldInferMappingModelAutomatically: shouldInferMappingModelAutomatically,
+                                            storeLoadCompletionHandler: { (store, error) in
+                                                storeLoadCompletionHandler(store + " (work)", error)
+                })
+
+            workContext = NSManagedObjectContext(concurrencyType: .mainQueueConcurrencyType)
+            workContext.persistentStoreCoordinator = workCoordinator
+            workContext.mergePolicy = mergePolicy
+
+            let backgroundCoordinator = NestedContextCoreDataStack
+                .persistentStoreCoordinator(withType: storeType,
+                                            storeName: storeName,
+                                            managedObjectModel: managedObjectModel,
+                                            shouldAddStoreAsynchronously: shouldAddStoreAsynchronously,
+                                            shouldMigrateStoreAutomatically: shouldMigrateStoreAutomatically,
+                                            shouldInferMappingModelAutomatically: shouldInferMappingModelAutomatically,
+                                            storeLoadCompletionHandler: { (store, error) in
+                                                storeLoadCompletionHandler(store + " (background)", error)
+                })
+
+            backgroundContext = NSManagedObjectContext(concurrencyType: .privateQueueConcurrencyType)
+            backgroundContext.persistentStoreCoordinator = backgroundCoordinator
+            backgroundContext.mergePolicy = mergePolicy
+
+            NotificationCenter.default.addObserver(self,
+                                                   selector: #selector(workContextChanged),
+                                                   name: .NSManagedObjectContextDidSave,
+                                                   object: workContext)
+
+            NotificationCenter.default.addObserver(self,
+                                                   selector: #selector(backgroundContextChanged),
+                                                   name: .NSManagedObjectContextDidSave,
+                                                   object: backgroundContext)
+        }
+    }
+
+    deinit {
+        NotificationCenter.default.removeObserver(self)
```

Comment:
🚔 You could remove this, since we are only supporting iOS9. But it's up to you 😄 😝